src/views/UserDetails.vue

```diff
@@ -1094,6 +1096,14 @@ export default defineComponent({
           showToast(translate("User status updated successfully."))
           await this.store.dispatch("user/updateSelectedUser", { ...this.selectedUser, ...payload });
           event.target.checked = isChecked
+          if (isChecked) {
+            this.blockLoginChecked = true;      
+              await UserService.updateUserLoginStatus({
```

Comment:
Block user login before disabling the party, because if we disable the party first then the block login operation on that party can't be performed.